.gitignore

```diff
@@ -3,3 +3,4 @@
 /doc
 erl_crash.dump
 *.ez
+.idea
```

Comment:
I wouldn't put editor files there. Everyone uses different those belong to users `~/.gitignore` file

Reply:
You're totally right ! This thing has bugged me for years, and yet I never thought about checking if a global .gitignore file was a thing. Thanks a lot, modified this.